docs/feature-gates.md

```diff
@@ -82,6 +83,13 @@ and will not implement Cluster IP functionality as expected.
 When using the OVS built-in kernel module (which is the most common case), your
 kernel version must be >= 4.6 (as opposed to >= 4.4 without this feature).
 
+### ServiceInternalTrafficPolicy
+
+`ServiceInternalTrafficPolicy` enables internal traffic restrictions to only route internal
+traffic to Endpoints within the Node the traffic originated from. The "internal" traffic
+here refers to traffic originated from Pods in the current cluster. This can help to reduce
+costs and improve performance.
```

Comment:
maybe add a link to the relevant K8s documentation

Reply:
Added.